src/abi/Admit.abi.ts

```diff
@@ -0,0 +1 @@
+export default [{"constant":false,"inputs":[{"name":"registry","type":"address"},{"name":"nft","type":"uint256"},{"name":"principal","type":"uint256"},{"name":"usr","type":"address"}],"name":"admit","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":false,"inputs":[{"name":"usr","type":"address"}],"name":"rely","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":false,"inputs":[{"name":"usr","type":"address"}],"name":"deny","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[{"name":"","type":"address"}],"name":"wards","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"},{"inputs":[{"name":"title_","type":"address"},{"name":"shelf_","type":"address"}],"payable":false,"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":false,"name":"loan","type":"uint256"}],"name":"Created","type":"event"}]
```

Comment:
Can we do this by importing the `abi` json file and afterwards exporting it?

Because if we have a new version of the contracts, we would have to update this file.
Correct?

Would be cool to just replace the abi.json file.